bin/picca_export_coadd_zint.py

```diff
@@ -109,7 +109,7 @@
     dmrp /= nbdm
     dmrt /= nbdm
     dmz /= nbdm
-elif dmrp.size==rp.size:
+if ('dmrp' not in locals()) or (dmrp.size==rp.size):
```

Comment:
Shouldn't that be an elif instead of if ?
If 'dmrp' is in locals(), and dmrp.size == rp.size, this will overwrite dmrp, dmrt, dmz

Reply:
We want to ovewrite dmrp if they have the same size has rp, since rp is computed on all the pairs in picca_cf, when dmrp is only computed with a fraction in picca_dmat.